schema/types/mountpoint.go

```diff
@@ -1,7 +1,57 @@
 package types
 
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
 type MountPoint struct {
 	Name     ACName `json:"name"`
 	Path     string `json:"path"`
 	ReadOnly bool   `json:"readOnly,omitempty"`
 }
+
+// MountPointFromString takes a command line mountpoint parameter and returns a mountpoint
+//
+// It is useful for actool patch-manifest --mounts
+//
+// Example mountpoint parameters:
+// 	database,path=/tmp,readOnly=true
+func MountPointFromString(mp string) (*MountPoint, error) {
```

Comment:
please add a test table for this? I haven't been hardline enough about people adding tests with schema code..